README_dev.md

```diff
@@ -101,6 +101,24 @@ Set the following environment variables:
 
 > We recommend to persist these variables for example in your `.bashrc` / `.zshrc` or in the Windows environment variables.
 
+### Local Backbone Prod Docker image testing
+
+To test the productive image you can use `docker compose -f .dev/compose.prodtest.yml --env-file .dev/compose.backbone.env`.
+
+For example to start the compose you can run it like
+`docker compose -f .dev/compose.prodtest.yml --env-file .dev/compose.backbone.env up --build -d`
+
+or to take it down again
+`docker compose -f .dev/compose.prodtest.yml --env-file .dev/compose.backbone.env down`
+
+to check if the prod image still works you can run.
+
+`docker logs -f connector`
+
+to see the logs of the connector.
+
+Afterward you can use the connector-tui or an REST client to test the connector.
```

Comment:
Some linguistic errors like "an REST client" instead of "a REST client" but not a big deal for now.